README.md

```diff
@@ -90,16 +92,53 @@ This command will format all files in the current directory and its subdirectori
 
 Please avoid committing code with formatting inconsistencies, as it helps maintain a clean and readable codebase.
 
-## Contributing
+Imports in `.tsx` files should follow the following order:
+
+- React/Next.js Imports
+- Library Imports
+- Type/Interface Imports
+- Other Component Imports
+- Styles or CSS Imports
+- Constants or Config Imports
+
+## Development Workflow & Contributing
 
-Contributions are welcome! If you'd like to contribute to this project, please make sure to follow the code formatting guidelines defined by Prettier. This ensures consistent code style and makes it easier to review and merge your contributions. If you'd like to contribute to the Musician Personnel Assignment Project, please follow these guidelines:
+Contributions are welcome! If you'd like to contribute to this project, please make sure to follow the code formatting guidelines defined by Prettier. This ensures consistent code style and makes it easier to review and merge your contributions. If you'd like to contribute to the Drum Roulette, please follow these guidelines:
```

Comment:
rogue "the" before Drum Roulette! After that is removed, I think ready for merging!

Reply:
@RubySpeeders changes are committed! Thanks for your review.

Reply:
FYI - After this is merged, I will then need to use the screenshot image URL from Github and add to the README.